src/main/java/servlets/YouTubeServlet.java

```diff
@@ -0,0 +1,109 @@
+/**
+ * Sample Java code for youtube.search.list
+ * See instructions for running these code samples locally:
+ * https://developers.google.com/explorer-help/guides/code_samples#java
+ */
+
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+
+import com.google.api.services.youtube.YouTube;
+import com.google.api.services.youtube.model.ResourceId;
+import com.google.api.services.youtube.model.SearchListResponse;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.Arrays;
+import java.util.Objects;
+import com.google.gson.Gson;
+
+@WebServlet("/videoResults")
+public class YouTubeServlet extends HttpServlet{
+    // Need Dev-Key to make an authorized search.
+    // For example: ... DEVELOPER_KEY = "YOUR ACTUAL KEY";
+    private static final String DEVELOPER_KEY = "AIzaSyD938GXrKFmO-z9VH49UJ7o04Z82iHJ8Rw";
```

Comment:
Can you update this using the method that Chris mentioned in the standup? (saving the key in a txt file and adding it to .gitignore) Or use another method to ensure that this key wouldn't be stolen since it's currently publicly available.

Reply:
Yes, I will put out another PR right after this one